jumanji/environments/logic/minesweeper/env.py

```diff
@@ -268,138 +254,39 @@ def _state_to_observation(self, state: State) -> Observation:
         return Observation(
             board=state.board,
             action_mask=jnp.equal(state.board, UNEXPLORED_ID),
-            num_mines=jnp.array(self.num_mines, jnp.int32),
+            num_mines=jnp.array(self.generator.num_mines, jnp.int32),
             step_count=state.step_count,
         )
 
-    def render(self, state: State) -> None:
-        """Render the given environment state using matplotlib.
-
+    def render(self, state: State) -> Optional[NDArray]:
+        """Renders the current state of the board.
         Args:
-            state: environment state to be rendered.
-
+            state: the current state to be rendered.
         """
-        self._clear_display()
-        fig, ax = self._get_fig_ax()
-        self._draw(ax, state)
-        self._update_display(fig)
+        return self._viewer.render(state=state)
 
     def animate(
         self,
         states: Sequence[State],
         interval: int = 200,
         save_path: Optional[str] = None,
     ) -> matplotlib.animation.FuncAnimation:
-        """Create an animation from a sequence of environment states.
-
-        Args:
-            states: sequence of environment states corresponding to consecutive timesteps.
-            interval: delay between frames in milliseconds, default to 200.
-            save_path: the path where the animation file should be saved. If it is None, the plot
-                will not be saved.
-
+        """Creates an animated gif of the board based on the sequence of states.
+         Args:
+        states: a list of `State` objects representing the sequence of states.
+        interval: the delay between frames in milliseconds, default to 200.
+        save_path: the path where the animation file should be saved. If it is None, the plot
+            will not be saved.
         Returns:
```

Comment:
Could we have an empty line between the description and Args? Similarly between the end of Args and Returns.